src/api/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:18.16.0
+
+# Create api directory
+WORKDIR /api
+
+# Install app dependencies
+# where available (npm@5+)
+COPY . .
+
+RUN npm install
+# If you are building your code for production
```

Comment:
You don't need these comments. Typically, a separate Dockerfile will be made for production or dev.